cbw_api_toolbox/cbw_api.py

```diff
@@ -26,13 +26,15 @@ def __init__(self, api_url, api_key, secret_key, verify_ssl=False):
     @staticmethod
     def _get_params(params=None):
         return "/{0}".format('/'.join(params)) if params else ""
-
-    def _request(self, route, params=None):
+    
+    def _request(self, route, verb, params=None):
         raw_response = None
         try:
-            raw_response = requests.get("{0}{1}{2}".format(self.api_url, route, self._get_params(params)),
-                                        auth=CBWAuth(self.api_key, self.secret_key),
-                                        verify=self.verify_ssl)
+            url = self.api_url + route + self._get_params(params)
```

Comment:
Please use format in place of concatenation.

https://softwareengineering.stackexchange.com/questions/304445/why-is-s-better-than-for-concatenation

Reply:
Please don't assign attribute like `verify = self.verify_ssl`

```python
raw_response = requests.request(verb,
                                "{0}{1}{2}".format(self.api_url, route, self._get_params(params)),
                                auth=CBWAuth(self.api_key, self.secret_key),
                                verify=self.verify_ssl)
```

Reply:
Done